Tropos/Controllers/TRWeatherController.m

```diff
@@ -37,6 +39,7 @@ - (instancetype)init
     self.geocodeController = [TRGeocodeController new];
     self.forecastController = [TRForecastController new];
     self.settingsController = [TRSettingsController new];
+    self.unitSystemChanged = [[self.settingsController unitSystemChanged] replayLastLazily];
```

Comment:
Does this need to be lazy? As in, is there a reason we wait for a subscription to open the multicast connection or is just for cleanliness?